torch/csrc/jit/tensorexpr/operators/quantization.cpp

```diff
@@ -598,11 +597,8 @@ Tensor computeQuantizedCat(
     const std::vector<ArgValue>& inputs,
     const std::vector<ExprHandle>& outputShape,
     const std::vector<ExprHandle>& outputStrides,
-    // NOLINTNEXTLINE
     const std::optional<ScalarType>& outputType,
-    // NOLINTNEXTLINE
     at::Device device) {
-  // NOLINTNEXTLINE(performance-unnecessary-copy-initialization)
   auto inputList = std::get<BufList>(inputs[0]);
```

Comment:
Not sure if this is actually a fix, or just no longer reported / not reported by the current clang-tidy version. At least I don't see why we need to make an explicit copy here when we only index into the list.

Reply:
Use reference